README.md

```diff
@@ -28,25 +28,35 @@ Documentation and guides are coming soon! In the mean time we suggest creating a
 
 ## To develop gRPC for ASP.NET Core
 
+Setting up local feed with unreleased Grpc.* packages (May not be necessary):
+```
+# We may depend on unreleased Grpc.* packages.
+# Run this script before building the project.
+./build/get-grpc.sh or ./build/get-grpc.ps1
+```
+
 Installing .NET Core SDK:
 ```
 # Run this script before building the project.
 ./build/get-dotnet.sh or ./build/get-dotnet.ps1
 ```
 
-Setting up local feed with unreleased Grpc.* packages:
+Activate the development environment:
 ```
-# We may depend on unreleased Grpc.* packages.
-# Run this script before building the project.
-./build/get-grpc.sh or ./build/get-grpc.ps1
+# Source this script to use the installed .NET Core SDK.
+source activate.sh or . activate.ps1
+```
+To launch Visual Studio with the installed SDK:
+```
+startvs.cmd Grpc.AspNetCore.sln
```

Comment:
We only have one solution. An argument is unnecessary.

Reply:
The script is set up to require an argument. Also, it can be used to open up *.csproj in addition to *.slns

Reply:
But in aspnetcore there are startvs.cmd scripts in solution directories that don't require an argument.

There is only one valid argument for startvs.cmd in this repo. Why are we making people do extra work?

Reply:
Hmm I was basing the behaviour off of https://github.com/aspnet/AspNetCore/blob/master/startvs.cmd but I suppose I can combine the functionality of that script and https://github.com/aspnet/AspNetCore/blob/master/src/SignalR/startvs.cmd with a default parameter.